chimney/src/main/scala/io/scalaland/chimney/internal/compiletime/derivation/transformer/rules/TransformProductToProductRuleModule.scala

```diff
@@ -96,7 +96,7 @@ private[compiletime] trait TransformProductToProductRuleModule { this: Derivatio
       val verifyNoOverrideUnused = Traverse[List]
         .parTraverse(
           filterOverridesForField(fromName =>
-            !parameters.keys.exists(toName => areNamesMatching(fromName, toName))
+            !parameters.keys.exists(toName => flags.fieldNameComparison.namesMatch(fromName, toName))
```

Comment:
I guess TransformerContext might introduce some utilities so that these would become `areFieldNamesEqual(fromName, toName)` where the utility would delegate to `flags.fieldNameComparison.namesMatch`.

Alternatively, to not require importing this utility everywhere could be:

```scala
def areFieldNamesMatching(fromName: String, toName: String)(
  implicit ctx: TransformationContext[?, ?]
): Boolean =
  ctx.config.flags.fieldNameComparison.namesMatch(fromName, toName)
```

defined in `Configurations`.

Reply:
Resolved on #478